test/base/RegistryManager.spec.ts

```diff
@@ -0,0 +1,45 @@
+import hre from "hardhat";
+import { loadFixture } from "@nomicfoundation/hardhat-toolbox/network-helpers";
+import { expect } from "chai";
+import { SignerWithAddress } from "@nomicfoundation/hardhat-ethers/signers";
+import { ZeroAddress } from "ethers";
+
+describe("RegistryManager", async () => {
```

Comment:
Should it test the happy path as well?

Reply:
update: Use ERC165 to check if valid interface id is supported when updating registry address.